test/framework/integration_framework/fake_peer/block_storage.hpp

```diff
@@ -25,33 +26,28 @@ namespace integration_framework {
 
     class BlockStorage final {
      public:
-      using BlockPtr = std::shared_ptr<shared_model::proto::Block>;
+      using BlockPtr = std::shared_ptr<const shared_model::proto::Block>;
       using HeightType = shared_model::interface::types::HeightType;
       using HashType = shared_model::crypto::Hash;
 
+      BlockStorage();
```

Comment:
Maybe pass `log` as a dependency, similar to `irohad` components?

Reply:
I would leave it for the new hierarchical logger, which is now in development.